packages/perseus-editor/src/editor.tsx

```diff
@@ -687,8 +686,7 @@ class Editor extends React.Component<Props, State> {
 
         const newContent = newPrelude + widgetContent + newPostlude;
 
-        const newWidgets = _.clone(this.props.widgets);
-        // @ts-expect-error TS(2345) Type '"categorizer" | undefined' is not assignable to type '"deprecated-standin"'.
+        const newWidgets = {...this.props.widgets};
         newWidgets[id] = {
             options: Widgets.getEditor(widgetType)?.defaultProps,
             type: widgetType as PerseusWidget["type"],
```

Comment:
We can remove this unsafe cast now!

```suggestion
            type: widgetType,
```